### ISSUE TITLE
Ansi Escape Code

### ISSUE THREAD

**Issue by dstadelm** [takaaki-kasai/git-foresta#8]
I'm trying to use this version of git-forest in combination with TamaMcGlinn/flog-forest in neovim. But the colors do not seem to work any more then. 
Are the colors printed as Ansi Escape characters? 
Is there anybody with experience with such a use case?

in the terminal
![image](https://user-images.githubusercontent.com/3010527/205022024-7cd341ca-f9c8-4468-8092-4aadb5715f01.png)

within neovim
![image](https://user-images.githubusercontent.com/3010527/205022394-6ef6aef8-628d-4cc4-81bb-531865a360b8.png)

**Comment by dstadelm** [comment]
Ok, I've found the issue within the flog-forest plugin

**Comment by TamaMcGlinn** [comment]
I built this plugin for this usecase:

```
Plug 'TamaMcGlinn/flog-forest'
```

**Comment by TamaMcGlinn** [comment]
Perhaps you need to install 'tklebanoff/vim-AnsiEsc' as well? If that fixes it, we should add this to the readme for flog-forest.

**Comment by dstadelm** [comment]
Indeed, and using the patched version of git-forest that is shipped with your plugin works just fine. However, I was wondering if it where possible to have colored branches. (and the style of this version git-foresta with the icons)

**Comment by dstadelm** [comment]
I've installed AnsiEsc and using your 'vanilla' plugin version works, just without colors for the branches
![image](https://user-images.githubusercontent.com/3010527/205025197-34b1b3c8-9432-4a2c-a07c-c94112b50975.png)

**Comment by TamaMcGlinn** [comment]
Probably possible, but that is issue #7 and I gave up trying to debug that. If you want to do that, you need to go into `~/.vim/plugged/flog-forest/git-forest`, checkout / cherry-pick the [experimental_ansi_colors](https://github.com/TamaMcGlinn/git-forest/tree/experimental_ansi_colors) branch and get it to work.

**Comment by dstadelm** [comment]
Thanks a lot for your incredible good work and insane fast response. If I find time I will definitely try that out.
